students/eldonab/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,54 @@
+# Task one
+formatted = 'file_{:03}: {:06.2f}, {:.2e}, {:.2e} '.format(2, 123.4567, 10000, 12345.67)
+print(formatted)
+
+#Task Two
+
+formatted_two_index = 'file_{0:03}: {1:06.2f}, {2:.2e}, {3:.2e} '.format(2, 123.4567, 10000, 12345.67)
+print(formatted_two_index)
+
+formatted_two = 'file_{ind_one:03}: {ind_two:06.2f}, {ind_three:.2e}, {ind_four:.2e}'.format(ind_one = 2, ind_two = 123.4567, ind_three = 10000, ind_four = 12345.67)
+print(formatted_two)
+
+
+# Task three
+# Rewrite: "the 3 numbers are: {:d}, {:d}, {:d}".format(1,2,3)" to take an arbitrary number of values.
+
+def formatter(t):
+    l = len(t)
+    listed = " ,".join(["{}"] * l)
+    my_str = "The numbers are: " + listed
+    return (my_str).format(*t)
```

Comment:
You'll want to include the count of numbers as well, as in the above note "the 3 numbers are...", etc.  You can use len() for that.